jupyter_ydoc/ybasedoc.py

```diff
@@ -27,7 +32,8 @@ def __init__(self, ydoc: Optional[Doc] = None):
         else:
             self._ydoc = ydoc
         self._ystate = self._ydoc.get("state", type=Map)
-        self._subscriptions: Dict[Any, Subscription] = {}
+        self._subscriptions = {}
+        self._undo_manager = UndoManager(self._ystate, 0)
```

Comment:
Currently `pycrdt` must pass an initial scope, but I don't think the `state` should be undoable. I'm not sure it's possible to not pass an initial scope in Yrs, see https://github.com/y-crdt/y-crdt/issues/457.
Also, passing `capture_timeout_millis=0` allows to undo every single update, but I don't know if it's desirable?

Reply:
With `pycrdt` v0.9.0 it is not necessary to pass an initial scope to an undo manager.